spec/lib/dor/text_extraction/cocina_updater_spec.rb

```diff
@@ -415,6 +436,11 @@
 
       it 'did not add a new file to the resource' do
         expect(dro.structural.contains[0].structural.contains.length).to eq 2
+        expect(dro.structural.contains[0].structural.contains[1].label).to eq 'page1.xml' # this is the first page
+        # the externalIdentifier and shas are the same as the original in our mocked cocina above
+        expect(dro.structural.contains[0].structural.contains[1].externalIdentifier).to eq "https://cocina.sul.stanford.edu/file/#{bare_druid}_001"
+        expect(dro.structural.contains[0].structural.contains[1].hasMessageDigests[0].attributes).to eq({ type: 'md5', digest: 'bogus_md5' })
+        expect(dro.structural.contains[0].structural.contains[1].hasMessageDigests[1].attributes).to eq({ type: 'sha1', digest: 'bogus_sha1' })
       end
```

Comment:
specs verifying we replace the file (or not)

these existing specs would have failed below since the file wasn't getting replaced cocina when it should have been